src/domain/Reward.ts

```diff
@@ -125,12 +143,12 @@ export function calculateRewards(date: BigInt, pool: Pool): void {
   systemRewards.save()
 }
 
-function checkRewardRate(checker: RewardDayTotal): void {
-  if (checker.toDateRewardAggregateValue.gt(BigDecimal.fromString(tierOneRewards))) {
-    log.debug('pending rewards > 1 MILL:  {}', [checker.toDateRewardAggregateValue.toString()])
+function setRewardRate(systemRewards: RewardDayTotal): RewardDayTotal {
+  if (systemRewards.toDateRewardAggregateValue.lt(BigDecimal.fromString(tierOneRewards))) {
+    log.debug('setting system rewards rate {}', [systemRewards.toDateRewardAggregateValue.toString()])
 
-    // TODO: update this with correct value
-    checker.rewardRate = BigDecimal.fromString('0')
-    checker.save()
+    systemRewards.rewardRate = BigDecimal.fromString('0.0042')
```

Comment:
@philipstanislaus this value is actually not correct as we need to multiply this by the number of decimals we have for RAD in substrate. It should be `0.0042*10**18` right?

Reply:
I have to double check, but on first glance 0.0042 sounds correct, since it should be a fraction.

Reply:
we checked this yesterday and it's correct. also working on some scripts to validate rewards amounts separately which are looking good 👍 still need to do the test claims on kovan and mainnet to fully validate